### DISCUSSION TITLE
Requirements

### DISCUSSION THREAD

**Discussion by davidwhicker** [spatie/laravel-pdf#12] [General]
You link to the here to install requirements on a forge server: https://spatie.be/docs/browsershot/v4/requirements

This is however trying to install node.js v14 with the latest being 20. Not sure if it cuases an issue but worth checking the whole requirement list?

**Comment by freekmurze** [comment]
You can use a higher node version if you want to.

Feel free to PR changes to the docs to make it more clear! 👍

**Comment by ultrono** [comment]
Quick one on this. Have just provisioned an Ubuntu 24.04 LTS server. `node` (`v18.20.3`) and `npm` (`10.7.0`) where both installed automatically.
